pallets/usage/src/tests.rs

```diff
@@ -112,6 +112,27 @@ fn wait_list_contents() -> Vec<(StoredDispatch, u32)> {
     .collect()
 }
 
+fn decode_validate_usage_call(encoded: &[u8]) -> crate::Call<Test> {
+    let mut encoded = <&[u8]>::clone(&encoded);
```

Comment:
Why not `encoded.clone()`?

Reply:
Clippy says so: clone of the reference is required here and clippy suggests to do it this way